resources/harmonizome/README.md

```
@@ -0,0 +1,166 @@
+    ora:
+      extra: "pvalueCutoff = 0.05, minGSSize = 10, maxGSSize = 500"
+    
+    # Parameters for GSEA analysis
+    gsea:
+      extra: "pvalueCutoff = 0.05, minGSSize = 10, maxGSSize = 500"
+```
+
+## File Format
+
+Gene sets are cached as RDS files with the following naming convention:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)